vcd/resource_vcd_independent_disk_test.go

```diff
@@ -175,3 +176,4 @@ resource "vcd_independent_disk" "{{.secondResourceName}}" {
   storage_profile = "{{.storageProfileName}}"
 }
 `
+*/
```

Comment:
I don't understand, why not leave this test, but only comment out the size_in_bytes field?

Reply:
cause it's duplicate

Reply:
It was skipped in the acceptance test but was still exported to the binary tests